src/osgPlugins/OpenFlight/Registry.h

```diff
@@ -1,13 +1,13 @@
-/* -*-c++-*- OpenSceneGraph - Copyright (C) 1998-2006 Robert Osfield
```

Comment:
This file appears to be keeping around a bunch of stuff that was removed in 3.6, are we sure we need all of the stuff we kept around?